set.mm

```diff
@@ -149722,19 +149722,20 @@ computer programs (as last() or lastChar()), the terminology used for
       VDRSZGZVFTFZNVFRSZGZVNVHPUTURVDUFFZVQACUGWAVOVPVDUHVDUIUJUMUTVFUFFZVTBDUG
       WBVRVSVFUHVFUIUJUMVDVFUKULUNURVEVIUTVGVJCUQQDUSQUOUP $.
 
-    $d x I $.
+    $d x A $.  $d x I $.
```

Comment:
It **is** required that `B` and `x` be disjoint. Simply, you do not see it in the diffs because the scope is too large. This is exactly the kind of trap I was advising to avoid in https://github.com/metamath/set.mm/pull/3760#discussion_r1451473596 :

> Hav[ing] large scopes makes it difficult to know which hypotheses are used/necessary where.

I stand by that remark.

Reply:
OK, `$d x B $.`is some lnes above. But the problem here was that in Github only the changed parts are shown (and a few lines before and after), no general problem with set.mm.

Reply:
> But the problem here was that in Github only the changed parts are shown (and a few lines before and after), no general problem with set.mm.

It's a bit like saying "the problem is not that this house has no stairs, the problem is that I cannot fly to the second floor".